dist/filelist

```diff
@@ -47,16 +47,16 @@ src/btree/col_srch.c
 src/btree/row_key.c
 src/btree/row_modify.c
 src/btree/row_srch.c
-src/checksum/arm64/crc32-arm64.c	ARM64_HOST
-src/checksum/loongarch64/crc32-loongarch64.c	LOONGARCH64_HOST
-src/checksum/power8/crc32_wrapper.c	POWERPC_HOST
-src/checksum/power8/vec_crc32.c		POWERPC_HOST
-src/checksum/riscv64/crc32-riscv64.c	RISCV64_HOST
+src/checksum/arm64/crc32-arm64.c    ARM64_HOST
+src/checksum/loongarch64/crc32-loongarch64.c    LOONGARCH64_HOST
+src/checksum/power8/crc32_wrapper.c    POWERPC_HOST
```

Comment:
It seems that we previously aligned the 2nd column and converting it to spaces has unaligned it, can we adjust with spaces?